packages/core/docs/core/use-user-shipping.md

```diff
@@ -0,0 +1,141 @@
+# useUserShipping composable
+
+<Content slot-key="header-warning" />
+
+`useUserShipping` composition function can be used to fetch, add new and modify existing user shipping addresses from the API.
+
+## Properties
+
+`useUserShipping` contains following properties:
+
+- `load` - function for fetching user addresses. When invoked, it requests data from the API and populates `shipping` property.
+
+- `addAddress` - function for posting new shipping address.
+
+<Content slot-key="add-params" />
+
+- `deleteAddress` - function for deleting existing shipping address.
+
+<Content slot-key="delete-params" />
+
+- `updateAddress` - function for updating existing shipping address.
+
+<Content slot-key="update-params" />
+
+- `setDefault` - function for settings an existing shipping address as default.
+
+::: tip
+If `isDefault` property is passed to `addAddress` or `updateAddress`, there is no need to call `setDefault` separately.
+:::
+
+<Content slot-key="set-default-params" />
+
+- `shipping` - reactive data object containing response from the backend.
+
+- `loading` - reactive object containing information about loading state of `load`, `addAddress`, `deleteAddress`, `updateAddress` and `setDefault` methods.
+
+## Getters
+
+Because `shipping` property is a raw response, it's recommended to use `UserShippingGetters` for accessing any data from it. It includes following helper functions:
+
+- `getAddresses` - returns list of shipping addresses.
```

Comment:
Below we have parts "from an individual address" parts everywhere. Maybe it would be user-friendly to add info about argument alsot here? I mean `shipping` as a arg

Reply:
There is the whole interface starting at line 81. Adding this here would be a unnecessary duplication